.cow.json

```diff
@@ -26,11 +26,10 @@
     "symbiote",
     "dnadesign"
   ],
-  "exclude": [
-    "silverstripe/recipe-plugin"
-  ],
   "upgrade-only": [
     "colymba/gridfield-bulk-editing-tools",
-    "tractorcow/silverstripe-fluent"
+    "tractorcow/silverstripe-fluent",
+    "silverstripe/recipe-plugin",
+    "silverstripe/vendor-plugin"
```

Comment:
Why have you added these to upgrade-only? We have control over these repositories so there's no need for that.

Reply:
I think it's probably ok to have as upgrade-only, I wrote it down in the original issue description

Looking at composer.json - recipe-plugin is a caret and vendor-plugin isn't even there, though it's a requirement of admin, also as a caret - https://github.com/silverstripe/silverstripe-admin/blob/2/composer.json#L25

I'm fine with them not being released as part of the big release, though we still want them in the changelog items, so upgarde-only is correct

Other option would be to put them in as explicit .x-dev requirements in composer.json and release them as normal modules.  Though we've already done stables for them - https://github.com/silverstripe/recipe-plugin/releases/tag/2.0.0 - https://github.com/silverstripe/vendor-plugin/releases/tag/2.0.0 - so for right now at least there's no need for this

Reply:
Fair enough. I'll assign back to you - I barely looked at this and I'm guessing you have a better background on it.